spec/controllers/expired_letters_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
```

Comment:
My preference would be to see this as a request spec instead of a controller spec, but ultimately 🤷‍♂️